lib/take_order.rb

```diff
@@ -0,0 +1,41 @@
+require 'rubygems' # not necessary with ruby 1.9 but included for completeness 
+require 'twilio-ruby'
+require './lib/order.rb'
+
+class Checkout
+
+  def initialize(money, total)
+    @money = money
+    @total = total#doesn't access order total, have to insert total
+    #upon initialising checkout
+  end
+
+def pay_money
+  if @money != @total
+    raise "You have entered an incorrect amount, please reenter"
+  else
+    puts "Thanks for your order! A text has been sent with your estimated delivery time."
+  end
+  send_message
+end
+
+ def delivery_time
+  @hourahead = Time.new + 3600
+ end
+ 
+  def send_message
+    delivery_time
+    account_sid = 'ACb869fa6a0e0d8bcbca82368079985d07' 
+    auth_token = '8ca316bf9c0c14d31739896ba2825085' 
+     
```

Comment:
Tokens are now published on github, so anyone can impersonate you on twilio. Avoid this by using environment variables. Google Dotenv gem for an easy way to do this.